cf-ssh.yml

```diff
@@ -5,6 +5,7 @@ applications:
   env:
     APPLICATION_HOST: dolores-app.18f.gov
     DEFAULT_URL_HOST: dolores-app.18f.gov
+    DISABLE_SANDBOX_WARNING: true
```

Comment:
we don't need this env var -- it was copied over from C2 but it isn't in use in this app